### DISCUSSION TITLE
Kobo - Koreader - set start up or favourite

### DISCUSSION THREAD

**Discussion by psyciknz** [koreader/koreader#9949] [Q&A]
Is there a way, or options, for setting the calibre libraries option to the main option on start up?   I use calibre for downloading the books.   But are not intending to use it for reading.   Preferring the stock reader. 

So my workflow would be 
- start koreader.
- navigate to calibre library
- download book
- exit koreader
- rescan to import downloaded books

So just wondering if there’s a way of getting the calibre library option surfaced from the search button to a man menu option .

TIA

**Comment by NiLuJe** [comment]
Not right now, no.

You might actually prefer an [UNCaGED](https://github.com/shermp/Kobo-UNCaGED/) setup for this kind of workflow (assuming you're fine with smart driver usage instead of OPDS/content server), since you wouldn't have to exit Nickel.

**Reply by psyciknz** [reply]
thanks had a look at that.  Unfortunately seemd to be driving from calibre (and doesn't support calibre-web).  So will stick with koreader at this point.

Hope we might be able to change some of the menu options in a future release, so i can lesson the number of clicks to access the opds library.
